front/docker-entrypoint.sh

```
@@ -1,7 +1,19 @@
 
-set -e
+set -eu
 
-rm -rf /usr/share/nginx/html/*
-cp -r /usr/local/src/ferriskey/* /usr/share/nginx/html
-envsubst < /usr/local/src/ferriskey/config.json > /usr/share/nginx/html/config.json
+HTML_DIR=/usr/share/angie/html
+SRC_DIR=/usr/local/src/ferriskey
+CONFIG_FILE="$HTML_DIR/config.json"
+
+rm -rf "$HTML_DIR"/*
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)